back/lib/tasks/single_use/services/file_upload_custom_field_migration_service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Tasks::SingleUse::Services::FileUploadCustomFieldMigrationService
+  def initialize
+    @stats = { ideas: 0, file_fields: 0, ideas_updated: 0, errors: [] }
+  end
+
+  attr_reader :stats
+
+  def migrate(persist_changes)
+    Rails.logger.info 'Migrating file upload format for idea custom fields.'
+    Rails.logger.info "Persist: #{persist_changes}"
+
+    custom_field_keys = CustomField.where(input_type: 'file_upload').pluck(:key)
```

Comment:
```suggestion
    custom_field_keys = CustomField.where(input_type: 'file_upload', resource_type: 'Idea').pluck(:key)
```
But it applies only if something else apart from `Idea` can have `file_upload` fields.

Reply:
Don't think anything else can have these custom fields, but doesn't harm to have it in there